server/src/constants/botId.js

```diff
@@ -0,0 +1,3 @@
+const botId = '5d326949e38a33679e223b07';
```

Comment:
Safe to put it here or shall we move it to the `.env` file?

Reply:
done